libraries/chain/include/eosio/chain/name.hpp

```diff
@@ -77,6 +77,19 @@ namespace eosio::chain {
       constexpr explicit operator bool()const { return value != 0; }
    };
 
+   namespace detail {
+      template <char... Str>
+      struct to_const_char_arr {
+         static constexpr const char value[] = {Str...};
+      };
+   } // namespace detail
+
+   template <typename T, T... Str>
+   inline name operator""_n() {
```

Comment:
This should be `constexpr`, I'm not sure why this is different than what is in eosio.cdt.  One thing is that you need to make sure the `name` constructors are `constexpr`.